pkgs/top-level/aliases.nix

```diff
@@ -1471,6 +1471,25 @@ mapAliases {
   lumail = throw "'lumail' has been removed since its upstream is unavailable"; # Added 2025-05-07
   lv_img_conv = throw "'lv_img_conv' has been removed from nixpkgs as it is broken"; # Added 2024-06-18
   lxd = lib.warnOnInstantiate "lxd has been renamed to lxd-lts" lxd-lts; # Added 2024-04-01
+
+  lxde = {
```

Comment:
Getting these eval errors may be confusing as it doesn't instruct you to look for `lxde.*`, how about rephrasing all the `throw`s to
```
foo = throw "'lxde.foo' has been moved to top-level, use 'foo' directly.";
```